openassessment/xblock/staff_area_mixin.py

```diff
@@ -76,6 +77,21 @@ def _wrapped(xblock, *args, **kwargs):  # pylint: disable=C0111
         return _wrapped
     return _decorator
 
+def conv_format_list(str_list):
```

Comment:
A few things:
1. The name is confusing to me. Recommend changing to `list_to_conversational_format` or something like that.
2. - Just my opinion (not an ask) -  I would just use python join and not bother with the method as while it's nice I don't think it adds much benefit.

Reply:
I agree on 1 and respectfully disagree on 2.  I tend to find it makes the UX feel more like you are interacting with something designed for people to have conversational lists.
Queried Sapana and UX and it sounds like there isn't a hard-opinion either way locally, but that conversational is nice when its easy.

Reply:
For the record: My preference would be having the frontend do the formatting and just passing a list forward to the template, but that is more annoying until a react framework is in place :-(